projects/netgrif-components-core/src/assets/i18n/sk.json

```diff
@@ -232,6 +232,8 @@
             "phone": "Zadaný text musí byť tel. číslo",
             "email": "Zadaný text musí byť email",
             "dateRange": "Zadaný dátum musí byť v rozsahu {{left}} a {{right}}",
+            "datePast": "Zadaný dátum musí byť v rozsahu do {{right}}",
+            "dateFuture": "Zadaný dátum musí byť v rozsahu od {{left}}",
```

Comment:
I believe `... musí byť pred/po` would be much clearer

Reply:
done